packages/uploadthing/src/next/core/approuter.ts

```diff
@@ -5,8 +5,8 @@ import {
 } from "../../internal/handler";
 import type { FileRouter } from "../../types";
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const UPLOADTHING_VERSION = require("../../../package.json").version;
+// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-member-access
+const UPLOADTHING_VERSION = require("../../../package.json").version as string;
```

Comment:
Not really related, but we should probably centralize this to a `consts.ts` file somewhere in the package.